src/main/java/nl/codefox/gilmore/command/criticalrole/CriticalRoleConstants.java

```diff
@@ -39,7 +39,7 @@
     public static final int TITLE_FONT_SIZE = 28;
     public static final int DATA_FONT_SIZE = 39;
 
-    public static final String SPREADSHEET_ID = "1uiweTErWOdoTuQThS6rt7V42YV4oRm9kVVzVBQsDCGI";
+    public static final String SPREADSHEET_ID = "1Pm8eLIYggq1vDMrR3EUjDOPlHqwC1owPbZj6rEgvm9M";

```

Comment:
Next time, do this in a separate branch and PR. Always keep PRs dedicated to one thing.